Package.swift

```diff
@@ -5,7 +5,7 @@ import PackageDescription
 
 // MARK: - Setup
 
-private let environment: Environment = .production
+private let environment: Environment = .development
```

Comment:
What's the reasoning here? 🤔

Reply:
With some other pull request by mistake we reversed this to production. So but we still are using only a development environment.